tests/foreman/cli/test_provisioning.py

```diff
@@ -40,3 +448,53 @@ def test_rhel_pxe_provisioning_on_libvirt():
 
     CaseLevel: Integration
     """
+    LOGGER.info(">>> User/pass: %s/%s" % user_credentials.auth)
+    LOGGER.info(">>> Org: [%s] %s" % (org.id, org.name))
+    LOGGER.info(">>> Loc: [%s] %s" % (loc.id, loc.name))
+    LOGGER.info(">>> Domain: [%s] %s" % (domain.id, domain.name))
+    LOGGER.info(">>> Subnet: [%s] %s" % (subnet.id, subnet.name))
+    LOGGER.info(">>> Lifecycle environment: [%s] %s" % (lce.id, lce.name))
+    LOGGER.info(">>> Base repo: [%s] %s" % (rhel7.id, rhel7.name))
+    LOGGER.info(">>> KS repo: [%s] %s" % (rhel7ks.id, rhel7ks.name))
+    LOGGER.info(">>> Content view: [%s] %s" % (cv.id, cv.name))
+    LOGGER.info(">>> Content view version: [%s]" % (cvv.id))
+    LOGGER.info(">>> Activation key: [%s] %s" % (ak.id, ak.name))
+    LOGGER.info(">>> Compute resource: [%s] %s" % (cr.id, cr.name))
+    LOGGER.info(">>> Architecture: [%s] %s" % (arch.id, arch.name))
+    LOGGER.info(">>> Operating system: [%s] %s %s.%s" % (os.id, os.name, os.major, os.minor))
+    LOGGER.info(">>> Environment: [%s] %s" % (env.id, env.name))
+    LOGGER.info(">>> Host group: [%s] %s" % (hg.id, hg.name))
+    host_ip = '192.168.11.104'   # FIXME Do not hardcode this one
```

Comment:
@jhutar I can still this IP here. Am I missing something?

Reply:
Sooory, fixing it now.